src/overlays/actors/ovl_Bg_Ikana_Mirror/z_bg_ikana_mirror.c

```diff
@@ -38,7 +38,7 @@ const ActorInit Bg_Ikana_Mirror_InitVars = {
     (ActorFunc)BgIkanaMirror_Draw,
 };
 
-static ColliderTrisElementInit sTrisElementsInit[9] = {
+static ColliderTrisElementInit sMirrorColliderElementsInit[9] = {
```

Comment:
Do you need to specify the number of elements here? Usually not, but static's sometimes weird about this.